### DISCUSSION TITLE
Does Rotz have category for linux distros ?

### DISCUSSION THREAD

**Discussion by aniketgm** [volllly/rotz#326] [General]
From the [examples](https://github.com/volllly/.dotfiles/blob/master/terminal/pwsh/dot.yaml), I can see `windows` and `linux` as separate category. However, I'd like to know if there is a category for different linux distros, like debian based, arch based, rpm based, etc.

If not, any plans for it in future ?

**Comment by volllly** [comment]
Hey, this is not implemented currently but we're discussing a possible aproach in https://github.com/volllly/rotz/issues/331. What do you think?

**Reply by aniketgm** [reply]
Looks great. As per your last comment in that thread. The `linux[whoami.distro^="Ubuntu"]:` approach looks more good, easy to follow and configure than the handlbar `{{ #if }}` statement.
